test/lint/lint-clang-format.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Unit-e developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Lint format strings: This program checks that the number of arguments passed
+# to a variadic format string function matches the number of format specifiers
+# in the format string.
+
+export LC_ALL=C
+
+python3 lint-clang-format.py
```

Comment:
Wouldn't it be better to directly call the python script from `lint-all.sh` instead of going through this kind of shell stub file?

Reply:
Hm, that's probably right. I somehow adapted this pattern which was presented here, as it seems to redirect to other python linters too, but there the shell scripts seem to be of some use actually, i.e. they do something. Here it's probably useless indeed.

Will change.

Reply:
Changed in https://github.com/dtr-org/unit-e/pull/1101/commits/3fcad73062ec180073bd442a9d1f364bc4b67051